WcaOnRails/app/views/database/_public_results_readme.md.erb

```diff
@@ -1,7 +1,9 @@
+<% long_date ||= DateTime.now %>
```

Comment:
I read that you should Time.current instead of (Date)Time.now ([here](https://discuss.rubyonrails.org/t/time-now-vs-time-current-vs-datetime-now/75183))

Reply:
I'm more confused than convinced by this article. We use a wild mix of `Time.now` and `DateTime.now` in our codebase, and both seem to work fine. In this case, the "worst" thing that could happen is that the timezone of some timestamp in the results export is off. Absolutely not critical and not worth investing more time into IMHO

Reply:
I got no hard feelings one way or the other, I did use `DateTime.current` in some of my code though if that is ok